src/components/MyAccount/RequestsTab/FreezeButton.tsx

```diff
@@ -29,7 +30,31 @@ const FreezeButton = ({ hold, patron }: { hold: Hold; patron: Patron }) => {
       <Heading className={styles.modalHeading}>
         <>
           <Icon size="large" name="errorFilled" color="ui.error.primary" />
-          <Text sx={{ marginBottom: 0 }}> Freezing this hold failed </Text>
+          <Text sx={{ marginBottom: 0 }}>
+            Hold {frozen ? "unfreeze" : "freeze"} failed
+          </Text>
+        </>
+      </Heading>
+    ),
+    onClose: () => {
+      closeModal()
+      setIsDisabled(false)
+    },
+  }
+  const successModalProps = {
+    type: "default",
+    closeButtonLabel: "OK",
+    headingText: (
+      <Heading className={styles.modalHeading}>
+        <>
```

Comment:
Very small nit but can you remove the fragments after the Heading components?

Reply:
The heading requires a single child component. I have a separate ticket to rearrange all of these components so the Icon is a child of the text component.